doc/examples/color_exposure/plot_hdr.py

```diff
@@ -0,0 +1,77 @@
+"""
+===========
+HDR Image
+===========
+
+A HDR (High Dynamic Range) image is a combination of bracketed images into one
```

Comment:
An HDR (High Dynamic Range) image combines several images of varying exposure into one.